w5e3.py

```diff
@@ -0,0 +1,31 @@
+"w5e3"
+#Parseltongue
+"""
+The course logo appears in the resources / logo.jpg file, and inside it are hidden some secret messages.
+The messages are strings at least 5 letters long, written in small English letters only and ending with an exclamation mark.
+Open the logo for reading in a binary configuration, and extract the secret messages from it.
+Keep in mind that the file may be very large, and it is best not to read it all in one go.
+Find help on the Internet regarding opening files in a binary form and a graded reading of the file.
+Be careful not to use techniques we have not learned (or add them only in addition to such a solution).
+"""
```

Comment:
When you right code you almost always want to include the logic you are writing in a function  - so it will be able to be used again somewhere else

Reply:
Not sure I understood the intent ..
To write in a more generic way?

Reply:
Yes, and coupling the input\output with the actual logic of the function is doing the opposite, and thats what you are doing in this function .
Read this to learn more - 
https://wiki.c2.com/?SeparateIoFromCalculation